.github/workflows/linux.yml

```diff
@@ -357,12 +357,6 @@ jobs:
         env:
           PYTHONPATH: "./build/:$PYTHONPATH"
 
-      - name: Test bindings (wheel)
-        run: |
-          source ${OV_INSTALL_DIR}/setupvars.sh
-          python -m pip install . --verbose --find-links ${OV_INSTALL_DIR}/wheels
-          python -m pytest ./tests/python_tests/test_whisper_generate_api.py
```

Comment:
why tests are removed?

Reply:
There's setupvars.sh based copy above

Reply:
I see. The above runs just one smoke test. I've added a filter to other to exclude it on the second run

Reply:
maybe we need to test only selected type of tests / use cases against build and all others against wheel?
E.g. Whisper is always testing against build, all other - wheel. In this case, we will cover both packages with no mixes smoke / no_smoke

Reply:
Tokenizers linkage is already tested by LLM tests. There's no need to design a general style for this. I just let whisper to do what it wants here